docs/azure/deployment.md

```diff
@@ -17,12 +17,12 @@ Whether your workflow is through the [Azure CLI](https://learn.microsoft.com/cli
 
 These tutorials from Microsoft Learn describe different ways of creating and deploying apps to Azure via Visual Studio Code:
```

Comment:
Should this sentence be tuned since only one of the links goes directly to a Microsoft Learn tutorial?
"The tutorials and topics below describe different ways of creating and deploying apps to Azure via Visual Studio Code:"